site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -131,6 +140,18 @@ const CreateConsequenceNetwork = (props: CreateConsequenceNetworkProps): ReactEl
                         <button className="govuk-button mt-8" data-module="govuk-button">
                             Save and continue
                         </button>
+
+                        {queryParams["return"]?.includes(REVIEW_DISRUPTION_PAGE_PATH) ? (
+                            <Link
+                                role="button"
+                                href={`${queryParams["return"] as string}/${pageState.disruptionId || ""}`}
```

Comment:
Im not sure what the || "" is going to do on the end of this line, because that will only be hit if the thing on the left of || is undefined, which it never will be.

Reply:
`disruptionId` is not a mandatory attribute for  generic `PageState` type. Hence the double quotes was added.

Reply:
Does that mean that our href will be whatever query params is, and then a '/' and nothing after it? Is that what we're wanting?

Reply:
If the disruptionId is empty then the page might fail before it reaches this line. If the condition is true then disruptionId will always be available. I have added an extra check to the condition to check if the id is defined